plugins/kafka-info/src/components/KafkaInfoComponent/KafkaInfoComponent.tsx

```diff
@@ -17,6 +22,21 @@ import { useEntity } from '@backstage/plugin-catalog-react';
 // These will let us get info about our backstage configuration
 import { useApi, configApiRef } from '@backstage/core-plugin-api';
 
+const clusterMap = [
+  stage: {
+    proxy: `kafka-lag-stage`,
+    name: 'Stage',
+    token: '${STAGE_PROMETHEUS_TOKEN}',
```

Comment:
This still looks wrong. It looks like you are either trying to do string interpolation incorrectly, or attempting an incorrect way of doing env var interpolation.

This also just doesn't make sense because you can put the token in the proxy by adding an auth header.